### PR TITLE
Additional anonymous usage stats

### PR DESCRIPTION
**Pull request by marefr**
Fixes #12056

### CODE REVIEW

File: pkg/services/sqlstore/sqlstore.go [hunk 1]
```diff
@@ -86,13 +86,13 @@ func (ss *SqlStore) Init() error {
 }
 
 func (ss *SqlStore) ensureAdminUser() error {
-	statsQuery := m.GetSystemStatsQuery{}
+	systemUserCountQuery := m.GetSystemUserCountStatsQuery{}
```
Comment (bergquist) [inline pkg/services/sqlstore/sqlstore.go]: :heart:

File: pkg/services/sqlstore/stats.go [hunk 2]
```diff
@@ -51,14 +60,32 @@ func GetSystemStats(query *m.GetSystemStatsQuery) error {
...
+		) as snapshots,
+		(
+			SELECT COUNT(id) FROM ` + dialect.Quote("team") + `
+		) as teams
 			`
 
 	activeUserDeadlineDate := time.Now().Add(-activeUserTimeLimit)
 	var stats m.SystemStats
-	_, err := x.SQL(rawSql, activeUserDeadlineDate).Get(&stats)
+	_, err := x.SQL(rawSql, activeUserDeadlineDate, dialect.BooleanStr(true), dialect.BooleanStr(false), dialect.BooleanStr(true)).Get(&stats)
```
Comment (bergquist) [inline pkg/services/sqlstore/stats.go]: I think we should declare those bool strings as variables or add them to the sql statement.

**Comment by torkelo** [comment]
nice!

File: pkg/services/sqlstore/stats.go (hunk 2 above)
Comment (marefr) [reply pkg/services/sqlstore/stats.go]: Pushed a change to use the sql builder instead.

**Pull request merged**